test/compflags/bradc/help/userhelp.good

```diff
@@ -22,6 +22,7 @@ Optimization Control Options:
       --[no-]dead-code-elimination    Enable [disable] dead code elimination
       --fast                          Use fast default settings
       --[no-]fast-followers           Enable [disable] fast followers
+      --[no-]force-initializers       Enable [disable] default initializers
```

Comment:
Don't you want to revert this one too, make --force-initializers be a developer only flag, and put tests for it specifically into test/deprecated?